### PR TITLE
Add Vue/React examples for Form - Validate and Submit and Configure Simple Items

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: concepts/05 Widgets/Form/05 Configure Simple Items/01 Create a Simple Item.md [hunk 1]
```diff
@@ -43,6 +43,68 @@ By default, the **Form** generates a simple item for each field of the [formData
...
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import Form from 'devextreme-react/form';
+
+    class App extends React.Component {
+        employee = {
```
Comment (jsdmitry) [inline concepts/05 Widgets/Form/05 Configure Simple Items/01 Create a Simple Item.md]: 'this' is lost

File: concepts/05 Widgets/Form/05 Configure Simple Items/01 Create a Simple Item.md [hunk 2]
```diff
@@ -112,6 +174,103 @@ The editor that will be used in a particular simple item depends on the type of
...
+    } from 'devextreme-react/form';
+
+    import Calendar from 'devextreme-vue/calendar';
+    import TextArea from 'devextreme-vue/text-area';
+
+    const calendarOptions = { value: new Date() };
+    const textAreaOptions = { placeholder: 'Add notes...' };
+
+    class App extends React.Component {
+        employee = {
```
Comment (jsdmitry) [inline concepts/05 Widgets/Form/05 Configure Simple Items/01 Create a Simple Item.md]: this is lost

File: concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md [hunk 3]
```diff
@@ -35,38 +36,161 @@ If none of the available editors suit your requirements, you can define a custom
...
+    	setNotes = (e) => {
+            this.setState(prevState => ({
+				employee: {
+                    ...prevState.employee,
+                    notes: e.value
+                }
+            }))
+        }
+        
+        renderNotes = (data) => {
```
Comment (jsdmitry) [inline concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md]: this is lost

File: concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md [hunk 4]
```diff
@@ -117,6 +241,83 @@ A simple item can be accompanied by a line of text that gives a hint, for exampl
...
+
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import Form, {
+        SimpleItem
+    } from 'devextreme-react/form';
+
+    class App extends React.Component {     
+        employee = {
```
Comment (jsdmitry) [inline concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md]: this is lost

File: concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md [hunk 5]
```diff
@@ -189,6 +390,97 @@ You can modify automatically generated items using the [customizeItem](/api-refe
...
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import Form from 'devextreme-react/form';
+
+    class App extends React.Component {     
+        employee = {
```
Comment (jsdmitry) [inline concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md]: this is lost

File: concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md [hunk 6]
```diff
@@ -189,6 +390,97 @@ You can modify automatically generated items using the [customizeItem](/api-refe
...
+            firstName: "John",
+            lastName: "Heart",
+            email: "jheart@dx-email.com",
+            phone: "+1(213) 555-9392"
+        }
+
+        customizeItem (item) {
```
Comment (jsdmitry) [inline concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md]:
```suggestion
        customizeItem(item) {
```

File: concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md [hunk 7]
```diff
@@ -189,6 +390,97 @@ You can modify automatically generated items using the [customizeItem](/api-refe
...
+            if (item.itemType == "simple") {
```
Comment (jsdmitry) [inline concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md]:
```suggestion
            if(item.itemType == "simple") {
```

File: concepts/05 Widgets/Form/40 Validate and Submit the Form.md [hunk 8]
```diff
@@ -61,6 +61,92 @@ The **Form** widget uses the built-in validation engine to validate form item va
...
+    import 'devextreme/dist/css/dx.light.css';
+
+    import Form, {
+        SimpleItem,
+        RequiredRule,
+        PatternRule
+    } from 'devextreme-react/form';
+
+    class App extends React.Component {
+        employee: {
```
Comment (jsdmitry) [inline concepts/05 Widgets/Form/40 Validate and Submit the Form.md]: this is lost

File: concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md [hunk 9]
```diff
@@ -35,38 +36,161 @@ If none of the available editors suit your requirements, you can define a custom
...
+            this.state = {
+         		employee: {
+                    name: "John Heart",
+                    picture: "https://js.devexpress.com/Content/images/doc/19_2/PhoneJS/person2.png",
+                    notes: "John has been in the Audio/Video industry since 1990. He has led DevAv as its CEO since 2003."
+                }       
+            }
+        }
+
+    	setNotes = (e) => {
```
Comment (jsdmitry) [inline concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md]:
```suggestion
    	const setNotes = (e) => {
```

File: concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md [hunk 10]
```diff
@@ -35,38 +36,161 @@ If none of the available editors suit your requirements, you can define a custom
...
+    class App extends React.Component {     
+        constructor(props) {
+            super(props);
+            this.state = {
+         		employee: {
+                    name: "John Heart",
+                    picture: "https://js.devexpress.com/Content/images/doc/19_2/PhoneJS/person2.png",
+                    notes: "John has been in the Audio/Video industry since 1990. He has led DevAv as its CEO since 2003."
+                }       
+            }
```
Comment (jsdmitry) [inline concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md]:
```suggestion
            };
```

File: concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md (hunk 9 above)
Comment (jsdmitry) [inline concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md]:
```suggestion
    	setNotes(e) {
```

File: concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md (hunk 3 above)
Comment (jsdmitry) [inline concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md]:
```suggestion
        renderNotes(data) {
```

File: concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md (hunk 10 above)
Comment (jsdmitry) [inline concepts/05 Widgets/Form/05 Configure Simple Items/05 Customize a Simple Item.md]:
```suggestion
            };
```

**Review by jsdmitry — APPROVED** [review]

**Review by AlexJameson — APPROVED** [review]

**Pull request merged**